Kong Studio Bundle Documentation (#1996)
Kong/docs.konghq.com: app/studio/1.0.x/deploy-to-dev-portal.md

### Introduction

By connecting to your Kong Enterprise instance, Kong Studio's Insomnia Designer 
Kong Plugin Bundle can sync Spec files to and from your Dev Portal.

### Prerequisites

> * [Insomnia Designer + Insomnia Designer Kong Plugin Bundle](/studio/{{page.kong_version}}/download-install)

Reviewer: ```suggestion
* [Insomnia Designer and the Insomnia Designer Kong Plugin Bundle] are installed (/studio/{{page.kong_version}}/download-install)
```

* Running Kong Enterprise 1.3 or later
* Admin API privileges for the Workspace you will be connecting to
* The Kong Developer Portal is enabled and running

### Step 1. Connect to Kong Enterprise

From Insomnia Designer:

> * Go to the Documents Listing View

Reviewer: ```suggestion
1. Open the Documents Listing View.
```

Author: Maybe here we use "Navigate to"

> * Find the spec you want to deploy to the Developer Portal

Reviewer: ```suggestion
2. Find the spec you want to deploy to the Developer Portal.
```

* Click on the documents **action menu** and select **Deploy to Kong Portal**

> ![Menu](https://s3.amazonaws.com/helpscout.net/docs/assets/59e383122c7d3a40f0ed78e2/images/5ea7f9d02c7d3a7e9aebbe6e/file-ZA5DLrBBPs.png)

Reviewer: Tab the image and the result (the prompt) to fall into the step.

Author: I wonder if this can also be done through the inline editor

> Insomnia Designer will prompt you for information to connect to your **Kong Admin API**

Reviewer: ```suggestion
Insomnia Designer will prompt you for information to connect to your **Kong Admin API**.
```

Author: Wasn't really sure on the periods at 3am

> * Enter your API URL

Reviewer: ```suggestion
4. Enter your API URL.
```

> * If your Admin API uses RBAC, enter the Token for to access the Workspace

Reviewer: ```suggestion
5. (Optional) If your Admin API uses RBAC, enter the Token for to access the Workspace.
```

Author: Nice

> ![Connect to Kong](https://s3.amazonaws.com/helpscout.net/docs/assets/59e383122c7d3a40f0ed78e2/images/5ea7fa1a2c7d3a7e9aebbe7d/file-aY1ixNgXnh.png)

Reviewer: tab this into the step

> * Click **Connect to Kong** to validate the connection. 

Reviewer: ```suggestion
6. Click **Connect to Kong** to validate the connection. 
```

### Step 2. Deploy a Spec file

> If successful, Insomnia Designer will then prompt you to specify a file name for the spec file - this will be the name of the file in the Developer Portal.

Reviewer: ```suggestion
If successful, Insomnia Designer will then prompt you to specify a name for the spec file &mdash; this will be the name that appears in the Developer Portal.
```

![Deploy Spec](https://s3.amazonaws.com/helpscout.net/docs/assets/59e383122c7d3a40f0ed78e2/images/5ea7fa7b04286364bc991bb9/file-jzWwnImfbw.png)

> * Click **Upload to Kong Portal** to upload the spec file

Reviewer: ```suggestion
1. Click **Upload to Kong Portal** to upload the spec file.
```

> *  If there is an existing file with the same name Insomnia Designer will prompt you to overwrite the existing file or cancel

Reviewer: ```suggestion
2.  If there is an existing file with the same name, Insomnia Designer will prompt you to overwrite the existing file or cancel.
```
